.gitignore

```diff
@@ -14,7 +14,7 @@ tsconfig.tsbuildinfo
 /fixtures/my-remix-app
 /fixtures/deno-app
 /integration/playwright-report
-/test-results
+test-results
```

Comment:
I noticed the test-results directory within integration was showing up in my git diffs.